projects/sw-routing/package.json

```diff
@@ -13,6 +13,11 @@
     "email": "jeffy@google.com",
     "url": "https://jeffy.info"
   },
+  "scripts": {
+    "lint": "eslint '.'",
```

Comment:
This is already handled in https://github.com/GoogleChrome/sw-helpers/blob/future-of-sw-tooling/gulpfile.js#L46